src/benchmarklib/benchmark_sql_executor.cpp

```diff
@@ -100,15 +100,28 @@ void BenchmarkSQLExecutor::_compare_tables(const std::shared_ptr<const Table>& e
   }
 }
 
-void BenchmarkSQLExecutor::_visualize(SQLPipeline& pipeline) const {
+void BenchmarkSQLExecutor::_visualize(SQLPipeline& pipeline) {
   GraphvizConfig graphviz_config;
   graphviz_config.format = "svg";
 
   const auto& lqps = pipeline.get_optimized_logical_plans();
   const auto& pqps = pipeline.get_physical_plans();
 
-  LQPVisualizer{graphviz_config, {}, {}, {}}.visualize(lqps, *_visualize_prefix + "-LQP.svg");
-  PQPVisualizer{graphviz_config, {}, {}, {}}.visualize(pqps, *_visualize_prefix + "-PQP.svg");
+  auto prefix = *_visualize_prefix;
+
+  if (_num_visualized_plans == 1) {
+    // We have multiple SQL pipelines in this benchmark item - rename the existing file
+    std::filesystem::rename(prefix + "-LQP.svg", prefix + "-0-LQP.svg");
```

Comment:
`== 1` signals multiple pipelines?

Reply:
Ah ... there's an increment. Ok ...

Reply:
Looks pretty strange, but ok.

Reply:
Any suggestions?

Reply:
Maybe "We have already visualized a prior SQL pipeline in this benchmark item"?

Reply:
Would be better. Probably not worth thinking too much about it.